spec/api-process-spec.js

```diff
@@ -38,16 +38,12 @@ describe('process module', () => {
     })
   })
 
-  // FIXME: Chromium 67 - getProcessMemoryInfo has been removed
-  // describe('process.getProcessMemoryInfo()', () => {
-  //   it('returns process memory info object', () => {
-  //     const processMemoryInfo = process.getProcessMemoryInfo()
-  //     expect(processMemoryInfo.peakWorkingSetSize).to.be.a('number')
-  //     expect(processMemoryInfo.privateBytes).to.be.a('number')
-  //     expect(processMemoryInfo.sharedBytes).to.be.a('number')
-  //     expect(processMemoryInfo.workingSetSize).to.be.a('number')
-  //   })
-  // })
+  describe('process.getMemoryFootprint()', () => {
+    it('resolves promise successfully with non zero value', async () => {
+      let memoryFootprint = await process.getMemoryFootprint()
+      expect(memoryFootprint).to.be.a('number').greaterThan(0)
+    })
+  })
```

Comment:
Needs a writeup in `docs/api/process.md`

Reply:
Yeah, mainly waiting for reviews on the API itself before writing the documentation.